index.js

```diff
@@ -15,12 +15,12 @@ global.AbortController = AbortController
     CosmJS
  */
 const mnemonic = process.env.MNEMONIC;
-console.log(process.env.MNEMONIC)
-const wallet = await DirectSecp256k1HdWallet.fromMnemonic(mnemonic);
+const DENOM = process.env.DENOM;
+const prefix = {prefix: process.env.PREFIX}
```

Comment:
```suggestion
const options = {prefix: process.env.PREFIX}
```

Reply:
We already have this variable called options 
https://github.com/noislabs/nois-bot/blob/9bf791bd87dd3e9c3d3e24fc37df18cb17f0fcd0/index.js#L39

Reply:
Let’s call it chain_options and drand_options